toponetx/__init__.py

```diff
@@ -21,6 +21,21 @@
     graph_2_clique_complex,
     graph_2_neighbor_complex,
 )
+from .utils.normalization import (
+    _compute_B1_normalized_matrix,
+    _compute_B1T_normalized_matrix,
+    _compute_B2_normalized_matrix,
+    _compute_B2T_normalized_matrix,
+    _compute_D1,
+    _compute_D2,
+    _compute_D3,
+    _compute_D5,
```

Comment:
These are internal functions (starting with an underscore) and should not be imported into the `toponetx` namespace.